src/components/organisms/PageHeader/ClusterSelectionTable.tsx

```diff
@@ -138,7 +139,13 @@ export const ClusterSelectionTable: FC<CLusterSelectionTableProps> = ({setIsClus
     if (clusterName === kubeConfigContext) {
       return;
     }
-    dispatch(setCurrentContext(clusterName));
+
+    runCommandInMainThread({
+      cmd: `kubectl config use-context ${clusterName}`,
```

Comment:
I think the cmd should be just 'kubectl', the following three args should be in the args array